prov/gni/src/gnix_cm.c

```diff
@@ -86,8 +86,8 @@ static int gnix_getname(fid_t fid, void *addr, size_t *addrlen)
 	/*
 	 * Retrieve the cdm_id & device_addr from the gnix_nic structure.
 	 */
```

Comment:
good point, I'll add type check and leave an assert for FI_EP_MSG.  We'll need something else for EP_MSG.  I believe we'll want to use an entirely different method for MSG list/connect/accept cm functionality.  I don't think the GNI postdata method is appropriate for that model.